src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -788,6 +788,11 @@
         <target state="new">Are you sure you want to delete selected URL wildcard(s)?</target>
         <note>key: url_wildcard.modal.message</note>
       </trans-unit>
+      <trans-unit id="7036f294d389a7a41226580ce5f0c62bc1a85c21" resname="user.setting.mode.name">
+        <source>user.setting.mode.name</source>
+        <target state="new">user.setting.mode.name</target>
+        <note>key: user.setting.mode.name</note>
+      </trans-unit>
```

Comment:
You are missing translation domain somewhere. `messages` domain shouldn't be used in product code.